broker/hcaxlsbroker.py

```diff
@@ -50,19 +50,23 @@
 
 v4_stringFields = {"donor" : ["age", "weight", "height"]}
 
-SCHEMA_URL = "https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/4.3.0/json_schema/"
-SCHEMA_URL = os.path.expandvars(os.environ.get('SCHEMA_URL', SCHEMA_URL))
+SCHEMA_URL = "https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/%s/json_schema/"
+# SCHEMA_URL = os.path.expandvars(os.environ.get('SCHEMA_URL', SCHEMA_URL))
+SCHEMA_VERSION = '4.4.0'
```

Comment:
Can you change this line to retrieve the schemas version from an env var, like in line 54?

Reply:
Sure, no problem